{
    "containers": {
        "cna": {
            "affected": [
                {
                    "product": "privoxy",
                    "vendor": "n/a",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "Privoxy 3.0.29",
                            "versionType": "custom"
                        }
                    ]
                }
            ],
            "descriptions": [
                {
                    "lang": "en",
                    "value": "A flaw was found in Privoxy in versions before 3.0.29. Memory leaks when a response is buffered and the buffer limit is reached or Privoxy is running out of memory can lead to a system crash."
                }
            ],
            "problemtypes": [
                {
                    "descriptions": [
                        {
                            "cweId": "CWE-401",
                            "description": "CWE-401",
                            "lang": "en",
                            "type": "CWE"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "dateUpdated": "2021-12-15",
                "orgId": "53f830b8-0a3f-465b-8143-3b8a9948e749",
                "shortName": "redhat"
            },
            "references": [
                {
                    "name": "https://bugzilla.redhat.com/show_bug.cgi?id=1928749",
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1928749"
                },
                {
                    "name": "https://www.privoxy.org/3.0.29/user-manual/whatsnew.html",
                    "url": "https://www.privoxy.org/3.0.29/user-manual/whatsnew.html"
                },
                {
                    "name": "GLSA-202107-16",
                    "url": "https://security.gentoo.org/glsa/202107-16"
                }
            ],
            "x_legacyV4Record": {
                "CVE_data_meta": {
                    "ASSIGNER": "secalert@redhat.com",
                    "ID": "CVE-2020-35502",
                    "STATE": "PUBLIC"
                },
                "affects": {
                    "vendor": {
                        "vendor_data": [
                            {
                                "product": {
                                    "product_data": [
                                        {
                                            "product_name": "privoxy",
                                            "version": {
                                                "version_data": [
                                                    {
                                                        "version_value": "Privoxy 3.0.29"
                                                    }
                                                ]
                                            }
                                        }
                                    ]
                                },
                                "vendor_name": "n/a"
                            }
                        ]
                    }
                },
                "data_format": "MITRE",
                "data_type": "CVE",
                "data_version": "4.0",
                "description": {
                    "description_data": [
                        {
                            "lang": "eng",
                            "value": "A flaw was found in Privoxy in versions before 3.0.29. Memory leaks when a response is buffered and the buffer limit is reached or Privoxy is running out of memory can lead to a system crash."
                        }
                    ]
                },
                "problemtype": {
                    "problemtype_data": [
                        {
                            "description": [
                                {
                                    "lang": "eng",
                                    "value": "CWE-401"
                                }
                            ]
                        }
                    ]
                },
                "references": {
                    "reference_data": [
                        {
                            "name": "https://bugzilla.redhat.com/show_bug.cgi?id=1928749",
                            "refsource": "MISC",
                            "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1928749"
                        },
                        {
                            "name": "https://www.privoxy.org/3.0.29/user-manual/whatsnew.html",
                            "refsource": "MISC",
                            "url": "https://www.privoxy.org/3.0.29/user-manual/whatsnew.html"
                        },
                        {
                            "name": "GLSA-202107-16",
                            "refsource": "GENTOO",
                            "url": "https://security.gentoo.org/glsa/202107-16"
                        }
                    ]
                }
            }
        }
    },
    "cveMetadata": {
        "assignerOrgId": "53f830b8-0a3f-465b-8143-3b8a9948e749",
        "assignerShortName": "redhat",
        "cveId": "CVE-2020-35502",
        "dateUpdated": "2021-12-15",
        "state": "PUBLIC"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.0"
}